src/main/python/fever/generate_subset.py

```diff
@@ -0,0 +1,58 @@
+#
```

Comment:
Sorry, what's the use of this file?

Reply:
It generates the training data subset that I used to tune BM25 params on. Intent was to create a subset with balanced labels.